examples/text_labels_and_annotations/text_alignment.py

```diff
@@ -1,12 +1,48 @@
 """
-===================
-Precise text layout
-===================
+==============
+Text alignment
+==============
+
+Texts are aligned relative to their anchor point depending on the properties
+``horizontalalignment`` and ``verticalalignment``.
+
+.. plot::
```

Comment:
Why put this in a plot directive instead of in the example itself?

Reply:
Because we only care about the graphic, the code is irrelevant in this context. The plot directive hides the code.